### DISCUSSION TITLE
Autofill the input field with the remote numbers

### DISCUSSION THREAD

**Discussion by DRYN07** [rustdesk/rustdesk#10158] [General]
Hi,

Is it possible to autofill the input field by url or any other automation solution? 

Like https://rustesk.com/web/1234567, zapier, make.com

If this is not possible, then is this on the roadmap in the future?

If neither of them, then this is a feature-request :)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#uri--uni-link
